python/src/fastfec/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Provides the fastfec package"""
```

Comment:
I think the other `__init__.py` file can be removed (`python/src/__init__.py` — sorry to comment here; GitHub wouldn't let me comment an empty file)